src/js/cypress/Browser.ts

```diff
@@ -74,6 +80,30 @@ export class IframeBrowser {
     select(selector: string, value: string) {
         return this.#body.find(selector).select(value);
     }
+
+    getElement(selector: string, timeout: BrowserTimeout = "sm") {
+        return cy.get(selector, { timeout: this.getTimeoutTime(timeout) });
+    }
+
+    getElementText(selector: string) {
+        return this.#body.get(selector).invoke("text");
+    }
+
+    clickOnText(text: string, multiple = false) {
```

Comment:
1. This seems very fragile - how do we find the component that contains the text??
2. https://helpcenter.veeam.com/docs/styleguide/tw/click.html there's no `click ON`